src/arrayMethodPush.js

```diff
@@ -5,7 +5,11 @@
  */
 function applyCustomPush() {
   [].__proto__.push2 = function(...elements) {
-    // write code here
+    for (const index of elements) {
```

Comment:
are you sure it is really index?

Reply:
why nobody likes this word???))

Reply:
because it's not an index) index is used with for (let i = 0; ...) loop